node/out_response.js

```diff
@@ -126,7 +123,7 @@ TChannelOutResponse.prototype.sendCallResponseContFrame = function sendCallRespo
             }));
             break;
         case States.Streaming:
-            self.sendFrame.callResponseCont(args, isLast);
+            self._sendCallResponseCont(args, isLast);
```

Comment:
should we define these methods on the base class and make their body throw not implemented errors ?